pkgs/by-name/sh/show-midi/package.nix

```
@@ -0,0 +1,84 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, alsa-lib
+, freetype
+, libX11
+, libXrandr
+, libXinerama
+, libXext
+, libXcursor
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "show-midi";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "gbevin";
+    repo = "ShowMIDI";
+    rev = finalAttrs.version;
+    hash = "sha256-BtkfeHZyeSZH6wIojj3dd2nCS5R535dSWsis/hXJbPc=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [
+    alsa-lib
+    freetype
+    libX11
+    libXrandr
+    libXinerama
+    libXext
+    libXcursor
+  ];
+
+  enableParallelBuilding = true;
+
+  makeFlags = [
+    "-C"
+    "Builds/LinuxMakefile"
```

```suggestion
    "-C"  "Builds/LinuxMakefile"
```

P.S.: can't you try to set `makefile = Builds/LinuxMakefile`?

---

`"-C" "Builds/LinuxMakefile"` doesn't work well with the auto-formatter, but I can do `"-C Builds/LinuxMakefile"`.

I did try `makefile = "..."`, but it doesn't change the working directory, which is needed.